.github/workflows/llm-action-error-reporter.yml

```diff
@@ -15,10 +15,9 @@ jobs:
       actions: read
     steps: 
       - name: Analyze logs
-        uses: smartcontractkit/.github/actions/llm-action-error-reporter@5efdd03d4a3311e7a0954e5b3061dbb57596ca10 # v0.2.0
+        uses: smartcontractkit/.github/actions/llm-action-error-reporter@b565920d0948d70a74c9555791ecff002924af4d # v0.5.0
```

Comment:
I think this hash and this version (0.5.0) is for llm-pr-writer, not llm-action-error-reporter (0.3.0), which should be at `d125ca9fe5e3b410de7c6db4a4ce3ed7a0728cd6` still.

Reply:
Thanks, updated